### PR TITLE
add team.VerifyRoster(roster, signature, adminKeys)

### PR DESCRIPTION
**Pull request by paulfurley**

### CODE REVIEW

File: team/team_test.go [hunk 1]
```diff
@@ -84,9 +84,11 @@ name = "Kiffix"
 		})
 
 		t.Run("sets a valid signature", func(t *testing.T) {
-			verifyRosterSignature(t,
```
Comment (idrysdale) [inline team/team_test.go]: suspect you could also drop `func verifyRosterSignature()` now?

Comment (paulfurley) [reply team/team_test.go]: Ah great spot, I missed out that commit from this PR!

Comment (paulfurley) [reply team/team_test.go]: Actually I just forgot to delete it!

Comment (paulfurley) [reply team/team_test.go]: Done

**Comment by paulfurley** [comment]
rebased

**Pull request merged**